api/signhandler/signhandler.go

```diff
@@ -127,6 +139,27 @@ func (h *Handler) Handle(w http.ResponseWriter, r *http.Request) error {
 		return errors.NewBadRequestString("Unable to parse sign request")
 	}
 
+	if req.NotAfter.IsZero() {
```

Comment:
right now `req.NotAfter` / `req.NotBefore` will always be 0 as the user doesn't have the ability to update it via his POST but I still put that piece here for future implementation (in case in the future we want users to be able to post `req.NotAfter` / `req.NotBefore` values and we want `req.NotAfter` to have the priority over `Expiry`).